ophyd/__init__.py

```diff
@@ -7,3 +7,5 @@
 
 from .controls import *
 from .session import get_session_manager
+
+__version__ = 'v0.0.3.post0'
```

Comment:
Ask @tacaswell  … it is from his master!
On Feb 26, 2015, at 4:45 PM, Dan Allan <notifications@github.com<mailto:notifications@github.com>> wrote:


In ophyd/__init__.py<https://github.com/NSLS-II/ophyd/pull/71#discussion_r25467794>:

> @@ -7,3 +7,5 @@
>
>  from .controls import *
>  from .session import get_session_manager
> +
> +__version__ = 'v0.0.3.post0'


Why is .post0 hard coded here?

—
Reply to this email directly or view it on GitHub<https://github.com/NSLS-II/ophyd/pull/71/files#r25467794>.